### DISCUSSION TITLE
v0.1.0

### DISCUSSION THREAD

**Discussion by Yamakaky** [Yamakaky/gyromouse#26] [Release]
First public version !

Most simple features from JoyShockMapper are supported. See
`https://github.com/Yamakaky/gyromouse/blob/v0.1.0/src/config/all-settings-example`
for a list of all supported settings.

<hr /><em>This discussion was created from the release <a href='https://github.com/Yamakaky/gyromouse/releases/tag/v0.1.0'>v0.1.0</a>.</em>

**Comment by AbbanAli** [comment]
hey! the sensitivity is super low, how to increase it?

**Reply by Yamakaky** [reply]
That would be GYRO_SENS
